### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by warlockwario** [thecoder-001/MineColab#129]
This if statement is from the Make-a-Server section of the notebook
It looks like there's a typo just before the first elif
The previous line's print() isn't tabbed, which ends the if statement, and gets confused with the elif after

````
if server_type == "paper":
  a = requests.get("https://papermc.io/api/v2/projects/paper/versions/" + version)
  b = requests.get("https://papermc.io/api/v2/projects/paper/versions/" + version + "/builds/" + str(a.json()["builds"][-1]))
  print("https://papermc.io/api/v2/projects/paper/versions/" + version + "/builds/" + str(a.json()["builds"][-1]) + "/downloads/" + b.json()["downloads"]["application"]["name"])

  url = "https://papermc.io/api/v2/projects/paper/versions/" + version + "/builds/" + str(a.json()["builds"][-1]) + "/downloads/" + b.json()["downloads"]["application"]["name"]

  jar_name = "server.jar"
print('Downloading to Google Drive...')
elif server_type == "fabric":
  url = "https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.7.4/fabric-installer-0.7.4.jar"
  jar_name = "fabric-installer.jar"
elif server_type == "forge":
  url = f"https://maven.minecraftforge.net/net/minecraftforge/forge/{version}-{forge_version}/forge-{version}-{forge_version}-installer.jar"
  jar_name = "server.jar"
````
  
  TLDR:
  `print('Downloading to Google Drive...')` needs to be tabbed
